src/rabbit_mgmt_db_handler.erl

```diff
@@ -63,6 +70,7 @@ ensure_statistics_enabled() ->
             ok
     end,
     {ok, StatsLevel} = application:get_env(rabbit, collect_statistics),
+    rabbit_log:info("Using rates mode '~p'~n", [rates_mode()]),
```

Comment:
I'm not sure about that. It would be confusing to those who enable "full" management plugin (which is 99.9% of users).